src/internal/randutil/rand_test.go

```diff
@@ -23,3 +23,27 @@ func BenchmarkReader(b *testing.B) {
 		require.NoError(b, err)
 	}
 }
+
+func testLetters(t *testing.T, buf []byte) {
```

Comment:
This looks like it works probabilistically based on the size of buf. Maybe leave a comment about that?